test_unstructured/partition/pptx/test_files/README.md

```diff
@@ -1,3 +0,0 @@
-PPTX files in this directory are made by hand using PowerPoint. This most faithfully represents source files likely to be used by users of this library. Files produced by `python-pptx` are "cleaner" in certain respects and may pass tests that a "real" PowerPoint file would not.
```

Comment:
I added this directory back during my first couple weeks at Unstructured before I came to learn that test-files go in `/example-docs`. The one .pptx test file from here moved to `/example-docs` and this directory goes away (so this README it contained is no longer relevant).